src/launchpad/size/models/common.py

```diff
@@ -60,11 +60,12 @@ class FileInfo(BaseModel):
     model_config = ConfigDict(frozen=True)
 
     path: str = Field(..., description="Relative path in the bundle")
+    full_path: str = Field(..., description="Fully qualified path to the file")
```

Comment:
I was looking into this yesterday for a similar reason to yours. One thought: maybe this should use `exclude=True` so it doesn't get serialized later on? I think this can have a pretty big size effect on the resulting JSON file, and we only need the relative path for external use.